translations/German/general-notes.md

```diff
@@ -0,0 +1,82 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+* ~~"So'n Dreck kann jedem mal passieren."~~
+* ~~"Das ist die Seite, wo der Quelltext steht."~~
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in #Formal_or_Informal, stick to *Du*(informal)
+instead of *Sie*(formal) when addressing the reader.
+Always capitalize the *Du* when addressing him, do **not** use *du*.
+
+## Form of instruction
+
+**Prefer imperative over constructions with auxiliary verbs.**
+
+For instructions, try to use the imperative (e.g. *"Gehe auf die Seite"* -
+*"Go to the page"*) instead of constructions with auxiliary verbs
+(e.g. *"Du musst auf die Seite ... gehen"* - *"You have to go the page ..."*).
+This keeps the phrases short and avoids unnecessary addressings of the reader.
+
+## Form of titles
+
+**Use continuous titles with verbs in infinitive form**
+
+To be consistent with other online platforms, use continuous titles with verbs
+in infinitive form, e.g. *Manage streams* - *Kanle verwalten* instead of
+~~*Verwalte Kanle*~~.
+
+##Concatenation of words
+
+**Try to avoid them.**
+
+The German language is infamous for its concatenations of nouns
+(e.g. "Heizölrückstoßdämpfung", which means "fuel oil recoil attenuation").
+For the sake of correct rendering and simplicity, you should try to avoid such
+concatenations whenever possible.
+
+## Anglicisms
+
+**Use them if a millenial would understand them better than the German equivalent.**
+
+In contrast to other languages, German happily adapts modern words from other languages,
```

Comment:
I think it's more `Unlike other languages` - many languages happily adapt some words from some languages, but not from others, e.g. Muslim-majority countries happily adapt Arabic words, but not English.